### PR TITLE
Some documentation for the usage of the "abstract" element.

### PR DESCRIPTION
**Pull request by annando**

### CODE REVIEW

**Comment by tobiasd** [comment]
We have a "one sentence per line" rule for the docs.
Rendered it's the same, but git results in nicer diffs when changing text and one can better see all too long sentences which should be truncated in the middle.

File: doc/BBCode.md [hunk 1]
```diff
@@ -143,6 +143,62 @@ Map
...
+[abstract]Hi friends Here are my newest pictures![abstract]
+[abstract=twit]Hi my dear Twitter followers. Do you want to see my new 
+pictures?[abstract]
+[abstract=apdn]Helly my dear followers on ADN. I made sone new pictures 
+that I wanted to share with you.[abstract]
+Today I was in the woods and took some real cool pictures ...
+</pre>
+
+For Twitter and App.net the system will use the defined abstracts. For 
+other networks (e.g. when you are using the Statusnet connector) the
```
Comment (tobiasd) [inline doc/BBCode.md]: it's GNU Social I think

File: doc/de/BBCode.md [hunk 2]
```diff
@@ -145,6 +145,64 @@ eine Karte von [OpenStreetmap](http://openstreetmap.org) eingebettet werden. Zur
...
+<pre>
+[abstract]Hallo Leute, hier meine neuesten Bilder![abstract]
+[abstract=twit]Hallo Twitter-User, hier meine neuesten Bilder![abstract]
+[abstract=apdn]Hallo App.net-User, hier meine neuesten Bilder![abstract]
+Ich war heute wieder im Wald unterwegs und habe tolle Bilder geschossen ...
+</pre>
+
+Für Twitter und App.net nimmt das System die entsprechenden Texte. Bei 
+anderen Netzwerken, bei denen der Inhalt gekürzt wird (z.B. beim 
+Statusnet-Connector) wird dann die Zusammenfassung unter [abstract]
```
Comment (tobiasd) [inline doc/de/BBCode.md]: same here, not StatusNet but GNU Social

File: doc/BBCode.md (hunk 1 above)
Comment (annando) [inline doc/BBCode.md]: The connector still is named "statusnet". I know that it is confusing.

Comment (tobiasd) [inline doc/BBCode.md]: Ok

File: doc/BBCode.md [hunk 3]
```diff
@@ -143,6 +143,47 @@ Map
...
+<pre>
+[abstract]Hi friends Here are my newest pictures![abstract]
+[abstract=twit]Hi my dear Twitter followers. Do you want to see my new 
+pictures?[abstract]
+[abstract=apdn]Helly my dear followers on ADN. I made sone new pictures 
+that I wanted to share with you.[abstract]
+Today I was in the woods and took some real cool pictures ...
+</pre>
+
+For Twitter and App.net the system will use the defined abstracts. For other networks (e.g. when you are using the Statusnet connector) the general abstract element will be used.
```
Comment (tobiasd) [inline doc/BBCode.md]: these are 2 sentences

File: doc/BBCode.md [hunk 4]
```diff
@@ -143,6 +143,47 @@ Map
...
 This require "openstreetmap" addon version 1.3 or newer.
 
+-----------------------------------------------------------
+
+Abstract for longer posts
+-------------------------
+
+If you want to spread your post to several third party networks you can have the problem that these networks have (for example) a length limitation. (Like on Twitter)
+
+Friendica is using a semi intelligent mechanism to generate a fitting abstract. But it can be interesting to define an own abstract that will only be displayed on the external network. This is done with the [abstract]-element. Example:
```
Comment (tobiasd) [inline doc/BBCode.md]: here are 3 sentences ;-)

File: doc/BBCode.md [hunk 5]
```diff
@@ -143,6 +143,47 @@ Map
...
+
+<pre>[abstract]Totally interesting! A must-see! Please click the link![/abstract]
+I want to tell you a really boring story that you really never wanted 
+to hear.</pre>
+
+Twitter would display the text "Totally interesting! A must-see! Please click the link!". On Friendica you would only see the text after "I want to tell you a really ..."
```
Comment (tobiasd) [inline doc/BBCode.md]: here are 2 sentences

File: doc/BBCode.md [hunk 6]
```diff
@@ -143,6 +143,55 @@ Map
...
+<pre>
+[abstract]Hi friends Here are my newest pictures![abstract]
+[abstract=twit]Hi my dear Twitter followers. Do you want to see my new 
+pictures?[abstract]
+[abstract=apdn]Helly my dear followers on ADN. I made sone new pictures 
+that I wanted to share with you.[abstract]
+Today I was in the woods and took some real cool pictures ...
+</pre>
+
+For Twitter and App.net the system will use the defined abstracts. For other networks (e.g. when you are using the "statusnet" connector that is used to post to GNU Social) the general abstract element will be used.
```
Comment (tobiasd) [inline doc/BBCode.md]: after "abstracts" there is a sentence complete.

**Comment by tobiasd** [comment]
I don't like the online editor, it never really worked for me. Especially with large files (boot.php IIRC) it truncated the file sowhere in the middle...

**Pull request merged**